sidebars.js

```diff
@@ -956,73 +956,73 @@ module.exports = {
                     label: 'Sauce Connect Proxy',
                     collapsed: true,
                     items: [
+                        'secure-connections/sauce-connect-5',
+                        'secure-connections/sauce-connect-5/installation',
+                        'secure-connections/sauce-connect-5/quickstart',
                         {
                             type: 'category',
-                            label: 'Sauce Connect Proxy 5',
+                            label: 'Setup and Configuration',
+                            collapsed: true,
+                            items: [
+                                'secure-connections/sauce-connect-5/operation/overview',
+                                'secure-connections/sauce-connect-5/operation/configuration',
+                                'secure-connections/sauce-connect-5/operation/proxies',
+                                'secure-connections/sauce-connect-5/operation/monitoring',
+                                'secure-connections/sauce-connect-5/operation/docker',
+                                'secure-connections/sauce-connect-5/operation/kubernetes',
+                                'secure-connections/sauce-connect-5/operation/systemd',
+                            ],
+                        },
+                        {
+                            type: 'category',
+                            label: 'Advanced',
                             collapsed: true,
                             items: [
-                                'secure-connections/sauce-connect-5',
-                                'secure-connections/sauce-connect-5/installation',
-                                'secure-connections/sauce-connect-5/quickstart',
+                                'secure-connections/sauce-connect-5/advanced/architecture',
+                            ],
+                        },
+                        {
+                            type: 'category',
+                            label: 'Sauce Connect Proxy 4',
+                            collapsed: true,
+                            items: [
+                                'secure-connections/sauce-connect',
```

Comment:
By leaving these as SC4 docs, anything external is going to keep pointing here. If we switch to `sauce-connect-4` for the old ones and move `sauce-connect-5` to `sauce-connect`, they'll get the new ones.

Reply:
I think we discussed and decided to not break sc-4 links for now and leave them as is.  We should probably update the links as well later but this MR changes the sidebar only.